_reposense/config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "authors":
+    [
+    {
+      "githubId": "Hilda-Ang",
+        "displayName": "HILDA...RAENI",
```

Comment:
Can't we use full names instead of redacted ones?

Reply:
They say must use from here [https://nus-cs2103-ay1819s1.github.io/cs2103-website/admin/reposenseConfigTemplates.html](url)

Reply:
Got it, thanks for the clarification.